src/client/HttpClient.ts

```diff
@@ -17,11 +17,9 @@ import {
  */
 export class HttpClient {
   protected readonly baseURL: string
-  protected readonly authToken?: string
```

Comment:
Bind a token to your client it is common and handy practice, why did you consider this not necesary?

Reply:
we currently do not use it, that's why I removed it. This is what we do:
```
 /**
   * Creates a new OrchestrateClient instance
   *
   * @param host - URL and port of the Orchestrate API
   */
  constructor(host: string) {
    this.client = new HttpClient({ host })
  }
```